Too many command line arguments. Configuration path expected - While running simple TF plan

We started seeing this error for the terraform plan within our orgs pipelines. It seems to have started happening with this PR was merged

https://github.com/microsoft/azure-pipelines-terraform/pull/36

```
/opt/hostedtoolcache/terraform/0.12.31/x64/terraform plan undefined -detailed-exitcode
Too many command line arguments. Configuration path expected.
##[error]Error: Terraform Plan failed with exit code: 1
Finishing: Terraform : plan
```

--- a/Tasks/TerraformTask/TerraformTaskV1/src/base-terraform-command-handler.ts
+++ b/Tasks/TerraformTask/TerraformTaskV1/src/base-terraform-command-handler.ts
@@ -115,11 +115,12 @@ export abstract class BaseTerraformCommandHandler {
     public async onlyPlan(): Promise<number> {
         this.warnIfMultipleProviders();
         let serviceName = `environmentServiceName${this.getServiceProviderNameFromProviderInput()}`;
+        let commandOptions = tasks.getInput("commandOptions") != null ? `${tasks.getInput("commandOptions")} -detailed-exitcode`:`-detailed-exitcode`
         let planCommand = new TerraformAuthorizationCommandInitializer(
             "plan",
             tasks.getInput("workingDirectory"),
             tasks.getInput(serviceName, true),
-            `${tasks.getInput("commandOptions")} -detailed-exitcode`
+            commandOptions
         );
         
         let terraformTool;
@@ -149,11 +150,12 @@ export abstract class BaseTerraformCommandHandler {
             const tempFileForPlanOutput = path.resolve(`temp-plan-${uuidV4()}.txt`);
 
             let serviceName = `environmentServiceName${this.getServiceProviderNameFromProviderInput()}`;
+            let commandOptions = tasks.getInput("commandOptions") != null ? `${tasks.getInput("commandOptions")} -out=${binaryPlanFilePath}`:`-out=${binaryPlanFilePath}`
             let planCommand = new TerraformAuthorizationCommandInitializer(
                 "plan",
                 tasks.getInput("workingDirectory"),
                 tasks.getInput(serviceName, true),
-                `${tasks.getInput("commandOptions")} -out=${binaryPlanFilePath}`
+                commandOptions
             );
             terraformTool = this.terraformToolHandler.createToolRunner(planCommand);
             this.handleProvider(planCommand);

--- a/Tasks/TerraformTask/TerraformTaskV2/src/base-terraform-command-handler.ts
+++ b/Tasks/TerraformTask/TerraformTaskV2/src/base-terraform-command-handler.ts
@@ -116,11 +116,12 @@ export abstract class BaseTerraformCommandHandler {
     public async onlyPlan(): Promise<number> {
         this.warnIfMultipleProviders();
         let serviceName = `environmentServiceName${this.getServiceProviderNameFromProviderInput()}`;
+        let commandOptions = tasks.getInput("commandOptions") != null ? `${tasks.getInput("commandOptions")} -detailed-exitcode`:`-detailed-exitcode`
         let planCommand = new TerraformAuthorizationCommandInitializer(
             "plan",
             tasks.getInput("workingDirectory"),
             tasks.getInput(serviceName, true),
-            `${tasks.getInput("commandOptions")} -detailed-exitcode`
+            commandOptions
         );
         
         let terraformTool;
@@ -150,11 +151,13 @@ export abstract class BaseTerraformCommandHandler {
             const tempFileForPlanOutput = path.resolve(`temp-plan-${uuidV4()}.txt`);
 
             let serviceName = `environmentServiceName${this.getServiceProviderNameFromProviderInput()}`;
+            let commandOptions = tasks.getInput("commandOptions") != null ? `${tasks.getInput("commandOptions")} -out=${binaryPlanFilePath}`:`-out=${binaryPlanFilePath}`
+            
             let planCommand = new TerraformAuthorizationCommandInitializer(
                 "plan",
                 tasks.getInput("workingDirectory"),
                 tasks.getInput(serviceName, true),
-                `${tasks.getInput("commandOptions")} -out=${binaryPlanFilePath}`
+                commandOptions
             );
             terraformTool = this.terraformToolHandler.createToolRunner(planCommand);
             this.handleProvider(planCommand);

